FlorenceBML/BMLacq681/BMLacq681.xml

```diff
@@ -35,9 +36,139 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                    </item> 
+                                        <item xml:id="q6" n="6">
+                                            <dim unit="leaf">3</dim>
+                                            <locus from="40r" to="42v"/>
+                                            s.l. 1, no stub,
+                                            s.l. 2, stub after 3,
+                                            s.l. 3, stub after 2.
+                                        </item> 
+                                        <item xml:id="q7" n="7">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="43r" to="52v"/>
+                                        </item> 
+                                        <item xml:id="q8" n="8">
+                                            <dim unit="leaf">10</dim>
+                                            <locus from="53r" to="62v"/>
+                                        </item> 
+                                        <item xml:id="q9" n="9">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="63r" to="70v"/>
+                                        </item> 
+                                        <item xml:id="q10" n="10">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="71r" to="78v"/>
+                                        </item> 
+                                        <item xml:id="q11" n="11">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="79r" to="86v"/>
+                                        </item> 
+                                        <item xml:id="q12" n="12">
+                                            <dim unit="leaf">9</dim>
+                                            <locus from="87r" to="95v"/>
+                                            s.l. 6, no stub
+                                        </item> 
+                                        <item xml:id="q13" n="13">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="96r" to="103v"/>
+                                        </item> 
+                                        <item xml:id="q14" n="14">
+                                            <dim unit="leaf">8</dim>
+                                            <locus from="104r" to="111v"/>
+                                        </item> 
+                                        <item xml:id="q15" n="15">
+                                            <dim unit="leaf">2</dim>
+                                            <locus from="112r" to="113v"/>
+                                        </item> 
+                                    </list>
```

Comment:
Great work :) 
Note that the closing tag `</list>` here is erroneously before the actual end of the list (after the last quire). This leads to error messages in the file and to the PR not passing the technical check.